README.md

```diff
@@ -42,6 +42,60 @@ To configure the harvester you have several harvester config options (in the har
 * `organization`: The organization to be associated to all harvested datasets (default: the organization, which owns the harvest source)
 
 
+## CLI Commands
+
+This extension provides a number of CLI commands to query/debug the results of the CSW server.
+
+
+### `search`
+
+To run a query an arbitrary query against the defined CSW server use the `search` command:
+
+```
+paster geocat search birds
+```
+
+This command takes an optional second parameter to specifiy the CSW url (defaults to `http://www.geocat.ch/geonetwork/srv/eng/csw`)
+
+### `cql`
+
+To run a query an arbitrary CQL query against the defined CSW server use the `cql` command:
```

Comment:
Something in this sentence sounds off, I mean duplicated :)

Reply:
Fixed.